spec/registrant_spec.rb

```diff
@@ -0,0 +1,63 @@
+# typed: ignore
+
+require "rspec"
+require "./lib/registrant"
+
+RSpec.describe Registrant do # rubocop:disable Metrics/BlockLength
+  before(:each) do
+    @registrant1 = Registrant.new({
+      name: "Bruce",
+      age: 18,
+      permit: true
+    })
+    @registrant2 = Registrant.new({
+      name: "Penny",
+      age: 15
+    })
+    @registrant3 = Registrant.new({
+      name: "Bobby",
+      age: 105,
+      permit: false,
+      license_data: {
```

Comment:
We wanted the license_data to just be a default value whenever the Registrant is initialized. So we don't even want the option to send this in as an argument. If we need to change the values, we can write a method to accomplish that.